src/storage/v3/id_types.hpp

```diff
@@ -35,6 +36,11 @@ namespace memgraph::storage::v3 {
     constexpr uint64_t AsUint() const { return id_; }                                                                 \
     constexpr int64_t AsInt() const { return std::bit_cast<int64_t>(id_); }                                           \
                                                                                                                       \
+    friend std::ostream &operator<<(std::ostream &in, const name &n) {                                                \
+      in << n.id_.AsInt();                                                                                            \
```

Comment:
```suggestion
      in << AsInt();                                                                                            \
```

`AsInt` is a member function of this class, not the `uint64_t` (as primitive types doesn't have any member functions).